tests/AulaTests.hs

```diff
@@ -161,3 +161,6 @@ boundary mn mx = frequency
     , (1, pure mx)
     , (98, choose (mn, mx))
     ]
+
+nonEmptyMarkdown :: Gen Document
+nonEmptyMarkdown = Markdown . (cs :: String -> ST) . getNonEmpty <$> arb
```

Comment:
I like that this is a special case of `nonEmpty`.  I guess it'd be nice to run pandoc here later, but even if we don't (which is my current preference) it's good to make that explicit in one place.